src/components/DateTimePicker/DateTimePicker.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import Picker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+
+const DateTimePicker = props => {
+  const { date, setDate, customInput, customInputProps, isTimeOnly, isSmallScreen, ...rest } = props;
+
+  const fixOnEnter = e => {
+    // makes sure input loses focus on enter, to avoid date parsing bugs
+    if (e.key.toLowerCase() === "enter") {
+      setTimeout(() => document.activeElement.blur());
+    }
+  };
+
+  const handleTimeChange = newTime => {
+    if (!newTime) return;
+    const dateWithOldTime = date || new Date();
```

Comment:
no, we need to make sure a proper date exists each time this function runs. what if we removed the date from another field (in case when we have two different fields for date and time), then we need a new date to set the time.